test/reline/test_config.rb

```diff
@@ -388,7 +388,8 @@ def test_additional_key_bindings_for_other_keymap
       "cd": "CD"
       set keymap emacs
       "ef": "EF"
-      set editing-mode vi # keymap changes to be vi-insert
+      # keymap changes to be vi-insert
+      set editing-mode vi
```

Comment:
Inline comment is not suggested according to this: 

https://www.linuxfromscratch.org/museum/blfs-museum/5.1/BLFS-5.1/postlfs/inputrc.html#:~:text=various%20options%20do.-,Note%20that%20comments%20can%20not%20be%20on%20the%20same%20line%20as%20commands.,-If%20you%20will

Reply:
Although inline comment does not exist, we should support ignore-after-space feature because readline behave like inline comment exists and it is actually used.
```inputrc
# ignore after space
"\C-f": history-search-forward ignored-string
set editing-mode vi ignored-string

# do not ignore after space
set emacs-mode-string emacs mode string
```
I think it is better to leave the test as is.

Reply:
Got it! Trimmed the comment before handling the line.